packages/apalis-sql/src/sqlite.rs

```diff
@@ -493,6 +522,44 @@ mod tests {
 
     use super::*;
     use email_service::Email;
+    use futures::StreamExt;
+    use std::ops::Sub;
+
+    /// migrate DB and return a storage instance.
+    async fn setup() -> SqliteStorage<Email> {
+        // Because connections cannot be shared across async runtime
+        // (different runtimes are created for each test),
+        // we don't share the storage and tests must be run sequentially.
+        let storage = SqliteStorage::<Email>::connect("sqlite::memory:")
+            .await
+            .expect("failed to connect DB server");
+        storage.setup().await.expect("failed to migrate DB");
+        storage
+    }
+
+    /// rollback DB changes made by tests.
+    /// Delete the following rows:
+    ///  - jobs whose state is `Pending` or locked by `worker_id`
+    ///  - worker identified by `worker_id`
+    ///
+    /// You should execute this function in the end of a test
+    async fn cleanup(storage: SqliteStorage<Email>, worker_id: String) {
+        let mut tx = storage
+            .pool
+            .acquire()
+            .await
+            .expect("failed to get connection");
+        sqlx::query("Delete from Jobs where lock_by = ?1 or status = 'Pending'")
+            .bind(worker_id.clone())
+            .execute(&mut tx)
+            .await
+            .expect("failed to delete jobs");
+        sqlx::query("Delete from Workers where id = ?1")
+            .bind(worker_id.clone())
+            .execute(&mut tx)
+            .await
+            .expect("failed to delete worker");
+    }
```

Comment:
Because sqlite is In-memory mode and pool is not shared among tests, there is no needs of rolling back DB.